apps/mull-api/src/app/event/location-autocomplete/location-autocomplete.resolver.ts

```diff
@@ -0,0 +1,14 @@
+import { Query, Args } from '@nestjs/graphql';
+import axiosInstance from '../../../../axiosConfig';
+
+export class LocationAutocompleteResolver {
+  @Query(/* istanbul ignore next */ () => [String])
+  async getAutocompletedLocations(@Args('userInput') userInput: String) {
+    const response = await axiosInstance.request({
+      method: 'get',
+      url: `https://app.geocodeapi.io/api/v1/autocomplete?apikey=${process.env.GEOCODE_KEY}&text=${userInput}&size=5`,
```

Comment:
The API doesn't seem to work too well, I put a park near my house and it couldn't find it. Could we look into a better API? Or is it another issue?

Very quick googling and I found this API which found the park near my house https://community.algolia.com/places/
Unfortunately they're shutting down in 2022, if we can't use this maybe there's others?

Reply:
Had the same issue as you and left a comment, have a look: https://github.com/RGPosadas/Mull/pull/121#discussion_r544754305

Reply:
The google places api pretty much gets my searches perfectly everytime, just like when I'm searching on google maps If we were to switch API I think google would be a good option

Reply:
changing the API might mean a lot of changes, for instance, the key might not have be in the backend because google has a way to restrict which domain can access the api. Also, with the frustrations mentioned above, fixing this ticket might not be worth it. If changing the API is necessary, then the ticket should just be restarted. Also, if we can change the implementation of the location autocomplete to not have a modal, but a searchable select instead, it will make this ticket so much easier. What do you think?